backend/app/project/routes.py

```diff
@@ -0,0 +1,18 @@
+from flask import Flask, jsonify, abort
+from . import project
+from . import db
+from app.models import Project
+import uuid
+
+# delete a project
+@project.route("/<project_uuid>", METHODS=["DELETE"])

```

Comment:
I think this has to be `@project.route("/<UUID:project_uuid>", METHODS=["DELETE"])`
but to simplify things I think `@project.route("/<UUID:id>", METHODS=["DELETE"])` will be good enough